requirements.txt

```diff
@@ -8,3 +8,10 @@ matplotlib
 scikit-learn
 ray[default]
 tokenizers==0.13.3
+### Extra dependency for Together
+#requests
+#sseclient-py
+### Extra dependency for Vertex AI
+#vertexai
```

Comment:
Actually the dependency for vertexai is confusingly: google-cloud-aiplatform

Reply:
You are right. fixing